internal/printer/printer.go

```diff
@@ -30,16 +31,21 @@ func init() {
 }
 
 func Error(w io.Writer, format string, args ...interface{}) error {
-	_, err := fmt.Fprintf(w, ErrorColor.Sprint("error: ")+format+"\n", args...)
-	return err
+	return writeOutput(w, ErrorColor.Sprint("error: ")+format+"\n", args...)
 }
 
 func Warning(w io.Writer, format string, args ...interface{}) error {
-	_, err := fmt.Fprintf(w, WarningColor.Sprint("warning: ")+format+"\n", args...)
-	return err
+	return writeOutput(w, WarningColor.Sprint("warning: ")+format+"\n", args...)
 }
 
 func Success(w io.Writer, format string, args ...interface{}) error {
-	_, err := fmt.Fprintf(w, SuccessColor.Sprint("✔ ")+fmt.Sprintf(format+"\n", args...))
+	return writeOutput(w, SuccessColor.Sprint("✔ ")+format+"\n", args...)
+}
+
+func writeOutput(w io.Writer, format string, args ...interface{}) error {
```

Comment:
do you mind if we don't do writeOutput and just do:

    \"%s\" 

at the call-site where we say "Switched [...]"? 

that way we don't have to fix and work around the color pkg. You can also a `// TODO` there saying `// TODO can't use %q there as the color sequences are escaped`

Reply:
I don't mind. I think that is the best way forward. The only downside is replacing all the `%q%` format strings currently in code.